test/runnable/traits.d

```diff
@@ -258,7 +258,45 @@ void test6()
 
 /********************************************************/
 
-void test7()
+void test7a()
+{
+    assert(__traits(isArray) == false);
+    assert(__traits(isArray, S) == false);
+    assert(__traits(isArray, C) == false);
+    assert(__traits(isArray, E) == false);
+    assert(__traits(isArray, void*) == false);
+    assert(__traits(isArray, void[]) == true);
+    assert(__traits(isArray, void[3]) == false);
```

Comment:
I think that this should pass, as a static array is still an array from a user's perspective. The fact that it decays to something else (a `struct`) is an implementation detail.